common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+trade_post = {
+	
+	############CULTURAL#################
+	
+	### General Building
+	tp_road_tavern = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { not_tradepost_culture_groups = yes }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		garrison_size = 0.5
+		tradevalue = 20
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	#Goblin Group
+	tp_zeppelin_tower = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = goblin_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 250
+		build_time = 180
+		ai_creation_factor = 95
+		
+		tech_growth_modifier = 0.125
+		military_techpoints = 0.01
+		tradevalue = 15
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game	
+	}
+	
+	##Nigth Elf Group
+	tp_resting_lounge = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = night_elf_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 225
+		build_time = 180
+		ai_creation_factor = 95
+		
+		monthly_character_piety = 0.03
+		garrison_size = 0.5
+		tech_growth_modifier = 0.125

```

Comment:
[The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tech_growth_modifier = 0.1
```
Plus, `retinuesize = 1` bonus is missing. Though it's too small. For example, `trade_offices_1` gives 50.